crates/sui-graphql-rpc/src/config.rs

```diff
@@ -530,11 +466,17 @@ impl Default for InternalFeatureConfig {
 impl Default for BackgroundTasksConfig {
     fn default() -> Self {
         Self {
-            watermark_update_ms: DEFAULT_WATERMARK_UPDATE_MS,
+            watermark_update_ms: 500,
```

Comment:
should we set this back to the `DEFAULT_WATERMARK_UPDATE_MS`? Or is this part of supporting the TS sdk shim layer

Reply:
DEFAULT_WATER_MARK_UPDATE_MS was 500, so the value hasn't changed. This PR removes the constants, so we need to inline them -- now there's a single source of truth for the default configs (the default impls of the relevant configs).